src/hooks/useAssets.ts

```diff
@@ -10,14 +10,23 @@ import { useCallback, useState } from 'react';
 
 import { useAccounts } from '@contexts/AccountsContext';
 
-import type { NativeTokenMetadata, TokenBalance, TokenMetadata } from '@helpers/interfaces';
+import type {
+  NativeTokenMetadata,
+  PalletDexPoolInfo,
+  PoolInfo,
+  TokenBalance,
+  TokenMetadata,
+} from '@helpers/interfaces';
+import type { PalletDexPoolId } from '@helpers/interfaces';
```

Comment:
looks like this type is a lone wolf
should join the others above though

Reply:
it's weird that the prettier doesn't join them automitically

Reply:
yep, thought about that, maybe there is a setting hmmm